### ISSUE TITLE
How can I setName to multiple fields?

### ISSUE THREAD

**Issue by daffoxdev** [Respect/Validation#719]
Hi! 
Can u help me please. Im trying to add name of fields to multiple rules using setName(), but looks like it not working.

Idea of code is something like this:
```php
... some code here
$input_data = [
    'user_name' => 'MyName111',
    'user_surname' => 'MySurname111',
    'user_tel' => 'asd123'
];
        
$this->rules = [
    v::key('user_name',     v::anyLetter())->setName('First Name'),
    v::key('user_surname',  v::anyLetter())->setName('Second Name'),
    v::key('user_tel',      v::phone())->setName('Phone number'),
];

try{
    v::allOf($this->rules)->setName('Validation Form')->assert($input_data);
} catch (ExceptionInterface $exception) {
    // ... setting translator. It works.
    
    $this->errors = $exception->findMessages(array_keys($input_data));
}
... some code here
```

This block not exactly what I used(I not tested it), because there I use layer class to add rules and other stuff, but the idea is same, generally I want to show idea.

If we will dump ```$this->errors```, we will see that {{name}} in errors will replaced by "user_name", "user_surname" etc., but not as I would expect: "First Name", "Second Name", etc.

This idea partly taken from this test code:
**respect\validation\tests\integration\find_messages_should_apply_templates_to_flattened_messages.phpt**

What I do wrong and is there possible to do what I need? Please help. 

PS. sorry for my English, maybe something I wrote grammatically uncorrecty.

**Comment by daffoxdev** [comment]
Looks like it is old bug https://github.com/Respect/Validation/issues/620 . Sadly that it not repaired still.

**Comment by henriquemoody** [comment]
This bug was finally fixed on versions [1.0.11](https://packagist.org/packages/respect/validation#1.0.11) and [1.1.6](https://packagist.org/packages/respect/validation#1.1.6) by #675.

I'm very sorry about my delay (I talked about that already https://github.com/Respect/Validation/issues/620#issuecomment-237474435).

Thanks for reporting! :panda_face:
